rust/private/rust.bzl

```
@@ -61,14 +61,14 @@ def _assert_correct_dep_mapping(ctx):
                 ),
             )
 
-def _determine_lib_name(name, crate_type, toolchain, lib_hash = ""):
+def _determine_lib_name(name, crate_type, toolchain, lib_hash = None):
     """See https://github.com/bazelbuild/rules_rust/issues/405
 
     Args:
         name (str): The name of the current target
         crate_type (str): The `crate_type`
         toolchain (rust_toolchain): The current `rust_toolchain`
-        lib_hash (str, optional): The hashed crate root path. Defaults to "".
+        lib_hash (str, optional): The hashed crate root path. Defaults to None.
```

No need to double document the default 😅 (this is my fault for using an auto-docs tool)
```suggestion
        lib_hash (str, optional): The hashed crate root path.
```